Date.spec.ts

```diff
@@ -65,6 +65,14 @@ describe("Date", () => {
 		expect(isoly.Date.lastOfMonth("2001-02-01")).toEqual("2001-02-28")
```

Comment:
What does Parity fixes mean? Please use a more descriptive title for the pull request.